doc/examples.md

```diff
@@ -48,3 +48,8 @@
     An example for a web server written with **servant-server** and
     [persistent](https://www.stackage.org/package/persistent) for writing data
     into a database.
+
+- [`import Servant` github search](https://github.com/search?l=Haskell&o=desc&q=%22import+Servant%22&s=indexed&type=Code)
+
+    It has thousands of results and can be a good way to see how people use servant in their projects or even to discover
```

Comment:
:O

Reply:
Yeah, I've been taking a look at this for a few years, regularly.

Reply:
I'd say there's no bad place for words like this. Thanks a lot!